datashuttle/utils/ds_logger.py

```diff
@@ -0,0 +1,114 @@
+import logging
+from pathlib import Path
+from typing import Any, List, Optional
+
+import fancylog as package
+from fancylog import fancylog
+from rich import print as rich_print
+from rich.console import Console
+from rich.filesize import decimal
+from rich.markup import escape
+from rich.text import Text
+from rich.tree import Tree
+
+from . import utils
+
+
+def start(
+    path_to_log: Path, name: str, variables: Optional[List[Any]]
+) -> None:
+    """"""
+    fancylog.start_logging(
+        path_to_log,
+        package,
+        filename=name,
+        variables=variables,
+        verbose=False,
```

Comment:
Would it be good to not hard code this to `False`? It may be useful to increase the amount of logging to the console when debugging.

Reply:
Sure so just to hard-code it to True? (or alternatively provide as option?)

Reply:
To use this option, I usually have some kind of "debug" mode in my code, that would set this to True, and also change some other settings (e.g. for data analysis s/w, saving intermediate files).

Reply:
That's a nice idea, for now I have exposed it as a top-level argument on `datashuttle._start_log()` and set default for True for now, as I guess we will be doing a lot of debugging. In future this can now be more easily extended to incorporate a debug mode.